packages/features/flags/config.ts

```diff
@@ -15,4 +15,5 @@ export type AppFlags = {
   "disable-signup": boolean;
   attributes: boolean;
   "organizer-request-email-v2": boolean;
+  "domain-wide-delegation": boolean;
```

Comment:
We might not need it now.

Reply:
team feature flag requires a featureSlug taken from Feature table

Reply:
Thanks!! makes sense